App.tsx

```diff
@@ -59,29 +61,32 @@ const App: React.FC = () => {
   });
 
   return (
-    <UserContext.Provider value={{ user, setUser, lastUsername, setLastUsername }}>
-      <BeneficiaryContext.Provider
-        value={{ current: beneficiary, setCurrent: setBeneficiary, list: beneficiaries, setList: setBeneficiaries }}>
-        <CenterContext.Provider value={{ list: centers, setList: setCenters }}>
-          <ContactContext.Provider value={{ list: contacts, setList: setContacts }}>
-            <NoteContext.Provider value={{ list: notes, setList: setNotes }}>
-              <EventContext.Provider value={{ list: events, setList: setEvents }}>
-                <DocumentContext.Provider value={{ list: documents, setList: setDocuments }}>
-                  <FolderContext.Provider value={{ list: folders, setList: setFolders }}>
-                    <ThemeContext.Provider value={{ value: theme, actions: themeActions }}>
-                      <NativeBaseProvider theme={MyTheme}>
-                        <StatusBar barStyle='light-content' />
-                        <Routes user={user} />
-                      </NativeBaseProvider>
-                    </ThemeContext.Provider>
-                  </FolderContext.Provider>
-                </DocumentContext.Provider>
-              </EventContext.Provider>
-            </NoteContext.Provider>
-          </ContactContext.Provider>
-        </CenterContext.Provider>
-      </BeneficiaryContext.Provider>
-    </UserContext.Provider>
+    <LoginTemporisationContext.Provider
+      value={{ attempts: attempts, setAttempts, isTemporarlyBlocked: isTemporarlyBlocked, setIsTemporarlyBlocked }}>
```

Comment:
You may make it simpler
`value={{ attempts, setAttempts, isTemporarlyBlocked, setIsTemporarlyBlocked }}>`